src/Sheet/Sheet.scss

```diff
@@ -0,0 +1,50 @@
+.pgn__sheet-skrim {
+  width: 100%;
+  height: 100%;
+  background-color: rgba(196,196,196,0.5);
+  position: fixed;
```

Comment:
We should probably specify a z-index. Here is where z-indexes are currently set: https://github.com/edx/paragon/blob/80ce916912cc13bafe81d2e90acb97b582cbb65f/scss/core/_variables.scss#L610-L611

We might want to add:
```
$zindex-sheet-backdrop:             1031 !default;
$zindex-sheet:                      1032 !default;
```

Reply:
reasonable.